api/metastore/v1/metastore.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package metastore.v1;
```

Comment:
nit: maybe we should put it in v2 or v2-experimental package

Reply:
This is the first version of metastore API.

As long as the API is not public, I see no practical benefits of separating the API